shared/src/libs/languages/en/index.ts

```diff
@@ -1,9 +1,11 @@
 import common from './common.json';
 import home from './home.json';
+import menuCreate from './menuCreate.json';
```

Comment:
better call it 'menu'
it will store all the localization related to menu page, not just create